tests/regression_tests/mgxs_library_condense/test.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, *args, **kwargs):
         self.mgxs_lib.by_nuclide = False
 
         # Test all MGXS types
-        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES + \
+        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES[:-1] + \
```

Comment:
Condensing a multigroup current to a coarser group structure is theoretically possible so it should be tested.

Reply:
The problem here is that the domain type imposed is 'material', but current can only work on a 'mesh' domain.

Reply:
Ok I'm fine with dropping that one. I can't think of a use case, CMFD current tallies should be done in the desired group structure from the get-go.